docker-compose.tests.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.4'
+
+networks:
+  integration-tests:
+    driver: bridge
+
+services:
+
+  vh-notification-api-tests:
+    image: notification-api-tests
+    build:
+      context: ./
+      dockerfile: tests/Dockerfile
+      args:
+        - PAT=$PAT_TOKEN
+    entrypoint:
+      [
+        "/wait-for-it.sh",
+        "test-db:1433",
+        "-t",
+        "120",
+        "--",
+        "/run-tests.sh"
+      ]
+    environment:
+      - ASPNETCORE_ENVIRONMENT=Development
+      - ConnectionStrings__VhNotificationsApi=Data Source=test-db;Database=VhNotificationsApi;User Id=SA;Password=R4ndomP@assword;Trust Server Certificate=True;Encrypt=False
```

Comment:
is there a way we can get these credentials from user secrets, or hash them out? Just so we can avoid having them in source control

Reply:
Why would you not want this in source control? Is it a security concern. These are safe to leave in source control because they're connection strings for the tests and nothing more. The appsettings.json already have a version of the connection string so it's no different to that.